tests/fixtures/optimizer/annotate_functions.sql

```diff
@@ -1631,6 +1631,18 @@ INT;
 LOWER(tbl.str_col);
 VARCHAR;
 
+# dialect: snowflake
+LTRIM('  hello world  ');
+VARCHAR;
```

Comment:
Why did the existing tests change / move? Let's avoid unnecessary deltas, it makes the review hard.